pkg/client/toolchain_object.go

```diff
@@ -97,3 +98,21 @@ func NewComparableToolchainObject(ob runtime.Object, compare CompareToolchainObj
 func (o *comparableToolchainObjectImpl) IsSame(otherObject ToolchainObject) (bool, error) {
 	return o.compare(o.ToolchainObject, otherObject)
 }
+
+func SortedComparableToolchainObjects(objects []ComparableToolchainObject) []ComparableToolchainObject {
```

Comment:
could you add a comment to explain that this function sorts the objects by their name, and maybe even rename it to something like `SortToolchainObjectsByName`?

Reply:
sure - done in 69cce42